File: Proj1/1A-Zawada-Mikolaj/GrpcGreeter/GrpcGreeter/MyData.cs
using System;
using System.Net;
using System.Net.Sockets;

namespace GrpcGreeter
{
    public class MyData
    {
        public static void info()
        {
            Console.WriteLine("Płóciennik Tomasz 260404");
            Console.WriteLine("Zawada Mikołaj 259431");
            Console.WriteLine(DateTime.Now.ToString("dd MMMM, HH:mm:ss", new System.Globalization.CultureInfo("pl-PL")));
            Console.WriteLine(Environment.Version);
            Console.WriteLine(Environment.UserName);
            Console.WriteLine(Environment.OSVersion);

            IPAddress[] localIPs = Dns.GetHostAddresses(Dns.GetHostName());
            foreach (IPAddress addr in localIPs)
            {
                if (addr.AddressFamily == AddressFamily.InterNetwork)
                {
                    Console.WriteLine(addr.ToString());
                }
            }
        }
    }
}

File: Proj5/jaxws1/src/main/java/org/example/jaxws/server_topdown2/MyData.java
package org.example.jaxws.server_topdown2;

import java.net.Inet4Address;
import java.net.InetAddress;
import java.net.UnknownHostException;
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class MyData {
    public static void info() {
        System.out.println("Płóciennik Tomasz 260404");
        System.out.println("Zawada Mikołaj 259431");
        LocalDateTime currentDateTime = LocalDateTime.now();
        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd MMMM, HH:mm:ss");
        System.out.println(currentDateTime.format(formatter));
        System.out.println(System.getProperty("java.version"));
        System.out.println(System.getProperty("user.name"));
        System.out.println(System.getProperty("os.name"));
        try {
            InetAddress localhost = InetAddress.getLocalHost();
            InetAddress[] localIPs = InetAddress.getAllByName(localhost.getHostName());
            for (InetAddress addr : localIPs) {
                if (addr instanceof Inet4Address) {
                    System.out.println(addr.getHostAddress());
                }
            }
        } catch (UnknownHostException e) {
            e.printStackTrace();
        }

    }
}
